.spelling

```diff
@@ -83,6 +83,7 @@ IPs
 IPv4
 Incrementality
 Initializers
+ISTIO_MUTUAL
```

Comment:
This shouldn't be in here, put backticks around the name of the variable in the text instead.

Reply:
Done